### PR TITLE
Include correlation coefficient in /report responses

### PR DESCRIPTION
**Pull request by wilsonianb**
Deduplicate total validations count
Order historical reports

### CODE REVIEW

**Comment by wilsonianb** [comment]
:+1:

File: api/services/ValidationReportService.js [hunk 1]
```diff
@@ -5,16 +5,31 @@ const DATE_FORMAT = 'YYYY-MM-DD'
 
 export async function historyForValidator(validationPublicKey) {
 
-  const reports = await database.ValidationReports.findAll()
+  const reports = await database.ValidationReports.findAll({
+    order: 'date DESC',
+    raw: true
```
Comment (stevenzeiler) [inline api/services/ValidationReportService.js]: Why `raw: true`?

Comment (wilsonianb) [inline api/services/ValidationReportService.js]: Raw gives the query result in json without any extra sequelize stuff. See "Raw Queries":
http://docs.sequelizejs.com/en/latest/docs/models-usage/

Comment (stevenzeiler) [inline api/services/ValidationReportService.js]: Ah I like that.

File: api/services/ValidationReportService.js [hunk 2]
```diff
@@ -56,8 +71,22 @@ export async function compute(start) {
...
+  })
+  const scores = await database.CorrelationScores.findOne({
+    where: {
+      date: report.date
+    },
+    raw: true
+  })
+  _.forEach(report.validators, (validations, validator)=>{
```
Comment (stevenzeiler) [inline api/services/ValidationReportService.js]: We will need to update the `webapp` to handle the new API format.

**Comment by stevenzeiler** [comment]
:+1:

File: api/services/ValidationReportService.js (hunk 2 above)
Comment (wilsonianb) [inline api/services/ValidationReportService.js]: I thought the webapp wasn't calling `/reports` yet, only `/reports/:validation_public_key`

**Pull request merged**